jwst/exp_to_source/tests/test_exp_to_source.py

```diff
@@ -3,8 +3,8 @@
 import numpy as np
 
 from . import helpers
-from ...datamodels import (MultiExposureModel, MultiSlitModel, ModelContainer)
-from ..exp_to_source import exp_to_source, multislit_to_container
+from jwst.datamodels import (MultiExposureModel, MultiSlitModel, ModelContainer)
```

Comment:
I thought the rule for this package was to due relative imports, rather than absolute.

Reply:
This is a test, and tests should have absolute imports.  That said, we have not been consistent here, so we have a bit of a mixed bag in our tests.